libgraphics/src/w32text.cpp

```diff
@@ -117,7 +119,7 @@ static bool w32_draw_text_process_transparent_buffer(uint32_t p_fill_color, bool
 			// If the destination pixel is opaque, then set it to transparent black
 			// and flag for opacity. Otherwise; take an average of the three RGB
 			// components and blend with the fill color.
-			if (((*t_context_pxl >> 24) & 0xFF) == 0xFF)
+			if (false && ((*t_context_pxl >> 24) & 0xFF) == 0xFF)
```

Comment:
The dead code gets removed in the update to use DirectWrite instead of GDI text rendering.